### PR TITLE
AAChen simulation

### PR DESCRIPTION
**Pull request by Jailander**
No topological map yet

### CODE REVIEW

**Comment by marc-hanheide** [comment]
Install targets?

**Comment by marc-hanheide** [comment]
I'll wait for the topo map as well.

**Comment by Jailander** [comment]
also added install targets, sorry I forgot

**Comment by Jailander** [comment]
@lucasb-eyer can you please check the topological map, you can insert it using `rosrun topological_utils load_yaml_map.py UMIC_FF.yaml`

**Comment by Jailander** [comment]
@lucasb-eyer if you want a simple tool to edit the map you can use this https://github.com/strands-project/strands_navigation/blob/indigo-devel/topological_navigation/launch/topological_map_edition.launch if it complains about transforms do this `rosrun tf static_transform_publisher 1 0 0 0 0 0 1 /map none 100` (I know I should add this to the launch file)

**Comment by lucasb-eyer** [comment]
why didn't you let me know when you started making the map, as we discussed?

Anyways, will test it soon.

**Comment by Jailander** [comment]
hmm I sent you an email

**Comment by Jailander** [comment]
no worries if you already have one I can use yours

**Comment by marc-hanheide** [comment]
So @lucasb-eyer have you prepared one already? I said @Jailander is working on it!

**Comment by Jailander** [comment]
As i said, its ok, I can adapt to @lucasb-eyer 's the only thing is that its quite urgent,

**Comment by lucasb-eyer** [comment]
nevermind, keep going with yours and I'll adapt it this evening or tomorrow morning, i.e. ASAP.

@marc-hanheide: I mean another mail from yesterday, i.e. before yours. But nevermind that now, I guess we misunderstood.
@Jailander I only got your mail asking about the charging point? To answer that one, though, it's at (0,0,0) in the slam-map.

**Comment by Jailander** [comment]
yes, I sent you one before from my phone it never left it sorry :( yes I noticed the Charging station was correctly positioned :) as I said you can use which ever version you like best, if you have any doubts on how to edit the map on yaml format or if you prefer to send me the old format so I'll work with it just email me

File: aachen/maps/UMIC_FF.yaml [hunk 1]
```diff
@@ -0,0 +1,3291 @@
+- meta:
+    map: UMIc_first_floor
```
Comment (lucasb-eyer) [inline aachen/maps/UMIC_FF.yaml:2]: Typo here, if this is case-sensitive.

Comment (Jailander) [inline aachen/maps/UMIC_FF.yaml:2]: yes, that field is still unused

**Comment by marc-hanheide** [comment]
Is this ready to be merges now?

**Comment by Jailander** [comment]
Yes, AFAIK the topological map was already wroking on site yesterday and the simulation works for me

**Comment by Pandoro** [comment]
Yes it worked. We tweaked it quite a bit but it was around 3 when we left.
Lucas eants to finish it today and upload the updates, but im not sure when
this will be exactly.

**Pull request merged**